ush/config.sh.RRFS_CONUS_3km_ens.2022HWT.FCST

```diff
@@ -195,6 +196,9 @@ fi
 RUN="RRFS_conus_3km"
 COMINgfs=""
 
+# PYTHON ENSEMBLE POSTPROC SETTINGS
+WGRIB2_IPOLATES="/misc/whome/rtrr/RAP/exec/UPP/wgrib2"
```

Comment:
Will we track the change history of this exe file?  Or we will build this exe from a repo?

Reply:
Please check if current system wgrib2 can do the interpolation. This is the only issue I have.

Reply:
Thanks for the input here.  Previously we did not have a wgrib2 module on Jet that could do this, but my knowledge was out of date.  Version 2.0.8 does it.  I have updated the module file (and put it in the correct place this time), removed any use of the external wgrib2 executable, and fixed some wording on the job script.